test/fail_compilation/test17908b.d

```diff
@@ -0,0 +1,15 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/test17908b.d(14): Error: function test17908b.foo is not callable because it is annotated with @disable
+---
+*/
+
+void foo();
+@disable void foo(int) {}
+alias g = foo;
```

Comment:
Could you also merge this file with the previous one, and add a body for `foo` ?

Reply:
On it.